ts/features/itwallet/machine/eid/machine.ts

```diff
@@ -466,6 +621,11 @@ export const itwEidIssuanceMachine = setup({
           target: "WalletInstanceRevocation"
         }
       }
+    },
+    SessionExpired: {
```

Comment:
We recently removed the `SessionExpired` state. Can you double-check?

Reply:
It was removed here: https://github.com/pagopa/io-app/pull/6371